mapStatusfunction.js

```diff
@@ -0,0 +1,33 @@
+array=[
+    {
+      number: 12
+    },
+    {
+      number: 3
+    },
+    {
+      number: 8
+    },
+    {
+      number: 14
+    },
+    {
+      number: 1
+    },
+    {
+      number: 7
+    }
+  ]
+function mapStatus(arr){

```

Comment:
Hi @NalamManasa 

Instead of using the `for` loop, please use the `.map` functions with the array.